tests/integration/main.js

```diff
@@ -821,6 +821,21 @@ test(`send messages [sends two messages to Bob's room]`, t => {
   t.timeoutAfter(TEST_TIMEOUT)
 })
 
+test(`check unread messages [Bob has 2 in Bob's room]`, t => {
+  const expectedRoomCounts = [(alicesRoom.id, 0), (bobsRoom.id, 2)].sort()
```

Comment:
tuples: not a thing!

```js
(x, y) === y
```

[god only knows what they were thinking with this one](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator)

just changing `(...)` to `[...]` should do the trick though

Reply:
:+1:

Reply:
changed it to filter on bobs room as we don't have a reference to the other room he's in.